### PR TITLE
seeds: speed up, more efficient and just less

### PR DESCRIPTION
**Pull request by mattscilipoti**
Utilizing DatabaseCleaner
Creating less seed examples

### CODE REVIEW

File: db/seeds/common.rb [hunk 1]
```diff
@@ -1,15 +1,4 @@
...
-Cohort.destroy_all
-Membership.destroy_all
-Observation.destroy_all
-Submission.destroy_all
-User.destroy_all
-Location.destroy_all
-Course.destroy_all
-Tagging.destroy_all
-Tag.destroy_all
+DatabaseCleaner.clean_with(:truncation)
```
Comment (RobertAKARobin) [inline db/seeds/common.rb]: Woooaaaaah.

**Comment by adambray** [comment]
:+1: :racehorse: :rocket: :speedboat:

**Comment by tylercrosse** [comment]
References issues #338 & #535

**Comment by nolds9** [comment]
:+1:

**Comment by jshawl** [comment]
:+1:

**Pull request merged**